packages/xchain-bitcoin/__mocks__/block-chair.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios'
+import MockAdapter from 'axios-mock-adapter'
+import { addresses, rawTransactions, dashboardTransactions, bitcoinStats } from './responses.json'
+
+const mock = new MockAdapter(axios)
+
+type MockConfig = {
+  url?: string
+}
+
+export default {
+  restore: mock.restore,
+  init: () => {
+    //Mock getAddress
+    mock.onGet(/\/dashboards\/address\//).reply(function (config: MockConfig) {
+      const id: any = config.url?.split('/').pop()
```

Comment:
Please avoid `any` here and at other places in the you code ...

Reply:
Fixed